### ISSUE TITLE
Explain what keptn is in plain English

### ISSUE THREAD

**Issue by kostis-codefresh** [keptn/keptn.github.io#198]
I am a software engineer and after reading 

- The main webpage
- The docs
- The announcement blog post

I still don't understand what keptn is exactly. The main webpage explains benefits in marketing speech. 

The "what is keptn" page just mentions a list of high level points but it still doesn't say what keptn is
For example: "The solution - keptn is fully serverless" doesn't really say anything about what keptn actually is.

The current information about keptn focuses too much on low-level concepts such as its architecture and its components but there is no information on what it actually does.

It should also be useful to have a clear overview on what tools keptn is reusing (e.g. Helm, git) and what brand new concepts it introduces (e.g. the shipyard files)

Finally the docs need a clear distinction on 
1) information for people who want to install keptn (i.e. operators, SREs)
2) information for people who want to use keptn as end users (i.e. developers)
3) information for people who want to extend keptn (advanced users)

Currently there is no such distinction.

**Comment by grabnerandi** [comment]
Hi Kostis. Thanks. Good feedback. As a starting point I have two resources but agree we need to update the docs:
* https://www.dynatrace.com/news/blog/what-is-keptn-how-it-works-and-how-to-get-started/
* https://www.youtube.com/watch?v=0vXURzikTac&list=PLqt2rd0eew1YFx9m8dBFSiGYSBcDuWG38&index=8&t=21s

Andi

**Comment by kostis-codefresh** [comment]
This repository that contains the specs https://github.com/keptn/keptn-specification should also be mentioned both in the home page as well as https://keptn.sh/docs/0.3.0/concepts/

**Comment by AloisReitbauer** [comment]
@kostis-codefresh  thank you for the feedback. I agree that the website should be much clearer. Most of the content is still from the early days when we decided to open source. We are currently reworking it and your feedback is spot on. Thank you for taking the time.

**Comment by danielkhan** [comment]
We made some improvements - I'd be happy to get your feedback @kostis-codefresh

**Comment by kostis-codefresh** [comment]
@danielkhan yes the main web page looks much better now. Thanks for taking care of this. The SRE/Dev/Devops tabs are very nice.

I think it would also be useful to add

* Some links to the shipyard, uniform file syntax explanation
* a link to the events supported by keptn (https://github.com/keptn/keptn-specification)
* a list of tools that keptn reuses (it is based on) so that people know that they can adapt their existing tools to be used in keptn (instead of thinking that they must throw everything away)

There are already several "boxes/cards" for these points in the main page. It is just that currently they are text only without actually linking to the "useful" content specification.

**Comment by christian-kreuzberger-dtx** [comment]
Hi @kostis-codefresh,

we've just had a major update of our keptn.sh landing page. We also introduced a page called "Why Keptn". 

Regarding links to events and the uniform syntax: The pages are available right now here: https://github.com/keptn/spec/
We still have some work to do with adding some hyperlinks to the respective spec and cloud events, but we aim to improve on that step by step (e.g., by adding some automation).

**Comment by kostis-codefresh** [comment]
Hello @christian-kreuzberger-dtx 

Yes I saw the updates. Looks good! As far as I am concerned this issue can be closed now.
